serverless-templates-gcp-application-python/README.md

```diff
@@ -0,0 +1,42 @@
+# Pulumi Workshop
+
+## Google Cloud Serverless Application in Python
+
+![Google Cloud Serverless Application](https://www.pulumi.com/templates/serverless-application/gcp/architecture.png)
+
+## Pulumi Fundamentals
+
+For this workshop, we're going to learn more about cloud computing by exploring how to use Pulumi to build, configure, and deploy your serverless application in Google Cloud using Python.
+
+
+We will create a storage bucket, an iam binding, a bucket object,  and a cloudfunction. Along the way, we'll learn more about how Pulumi works.
+
+## Goals
+
+* Build your first infrastructure with Pulumi
+* Learn fundamental Infrastructure as Code concepts
+* Learn fundamental Pulumi concepts
+* Build a serverless function in Google
+
+## Equipment, Tools, and Knowledge
+
+You will need the following tools to complete this workshop:
+
+[//]: # (Do we need to fix the url to have a referal link?)
```

Comment:
If you can, that would be lovely. I'll add the links as changes that you can accept if you'd like.

Reply:
Hi @nimbinatus,
Updated all the links